libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -340,9 +340,21 @@ void AP_AHRS::reset_gyro_drift(void)
  */
 void AP_AHRS::update_state(void)
 {
+    const uint8_t primary_gyro = _get_primary_gyro_index();
+    const uint8_t primary_accel = _get_primary_accel_index();
+#if AP_INERTIALSENSOR_ENABLED
+    // tell the IMUS about primary changes
+    if (primary_gyro != state.primary_gyro) {
+        AP::ins().set_primary_gyro(primary_gyro);
+    }
+
+    if (primary_accel != state.primary_accel) {
```

Comment:
i think we don't need to do anything with accel in this PR, we tie the accel and gyro index in AP_AHRS::_get_primary_IMU_index() now anyway

Reply:
Removed